roles/openshift_hosted_logging/tasks/deploy_logging.yaml

```diff
@@ -138,7 +138,7 @@
     delay: 10
 
   - name: "Deploy fluentd by labeling the node"
-    shell:  "{{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig label node {{ openshift_hostname }} {{ openshift_hosted_logging_fluentd_nodeselector if openshift_hosted_logging_fluentd_nodeselector is defined else 'logging-infra-fluentd=true' }}"
+    shell:  "{{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig label node --all {{ openshift_hosted_logging_fluentd_nodeselector_label if openshift_hosted_logging_fluentd_nodeselector_label is defined else 'logging-infra-fluentd=true' }}"
```

Comment:
At this time we don't offer a way for users to have the daemonset without the selector; you get our default if none is specified. I think we want a selector so that admins have a way to undeploy/redeploy fluentd without deleting the daemonset. I don't think (m)any users are going to care about changing the default selector. It seems unlikely to conflict.